### PR TITLE
Add swap

### PR DESCRIPTION
**Pull request by flgx16**
Displays swap space usage next to system RAM usage, requires `ram` to be enabled

### CODE REVIEW

File: src/hud_elements.cpp [hunk 1]
```diff
@@ -320,15 +320,24 @@ void HudElements::vram(){
...
+        ImGui::TableNextRow();
+        ImGui::TextColored(HUDElements.colors.ram, "RAM");
+        ImGui::TableNextCell();
+        right_aligned_text(HUDElements.colors.text, HUDElements.ralign_width, "%.1f", memused);
+        ImGui::SameLine(0,1.0f);
+        ImGui::PushFont(HUDElements.sw_stats->font1);
+        ImGui::Text("GiB");
+        ImGui::PopFont();
+    }
+
```
Comment (baryluk) [inline src/hud_elements.cpp:332]: Please remove unrelated code changes, including indenting.

Comment (flgx16) [reply src/hud_elements.cpp:332]: Done.

**Comment by baryluk** [comment]
Nice. Looks good to me!

+1
